packages/lib/translations/ro/marketing.po

```diff
@@ -0,0 +1,594 @@
+msgid "Careers"
+msgstr "Cariere"
+
+#: apps/marketing/src/components/(marketing)/footer.tsx:36
+msgid "Changelog"
+msgstr "Istoric de Modificări"
+
+#: apps/marketing/src/components/(marketing)/open-build-template-bento.tsx:81
+msgid "Choose a template from the community app store. Or submit your own template for others to use."
+msgstr "Alegeți un șablon din magazinul de aplicații al comunității. Sau trimiteți propriul șablon pentru ca alții să îl folosească."
+
+#: apps/marketing/src/app/(marketing)/open/page.tsx:219
+msgid "Community"
+msgstr "Comunitate"
+
+#: apps/marketing/src/app/(marketing)/open/monthly-completed-documents-chart.tsx:55
+msgid "Completed Documents"
+msgstr "Documente Finalizate"
+
+#: apps/marketing/src/app/(marketing)/open/monthly-completed-documents-chart.tsx:33
+msgid "Completed Documents per Month"
+msgstr "Documente Finalizate pe Lună"
+
+#: apps/marketing/src/components/(marketing)/share-connect-paid-widget-bento.tsx:61
+msgid "Connections"
+msgstr "Conexiuni"
+
+#: apps/marketing/src/components/(marketing)/enterprise.tsx:35
+msgid "Contact Us"
+msgstr "Contactați-ne"
+
+#: apps/marketing/src/components/(marketing)/share-connect-paid-widget-bento.tsx:63
+msgid "Create connections and automations with Zapier and more to integrate with your favorite tools."
+msgstr "Creați conexiuni și automatizări cu Zapier și altele pentru a integra cu instrumentele dvs. preferate."
+
+#: apps/marketing/src/components/(marketing)/call-to-action.tsx:23
+msgid "Create your account and start using state-of-the-art document signing. Open and beautiful signing is within your grasp."
+msgstr "Creați-vă contul și începeți să utilizați semnarea documentelor de ultimă generație. Semnarea deschisă și frumoasă este la îndemâna dvs."
+
+#: apps/marketing/src/app/(marketing)/open/tooltip.tsx:35
+msgid "Customers with an Active Subscriptions."
+msgstr "Clienți cu Abonamente Active."
+
+#: apps/marketing/src/components/(marketing)/open-build-template-bento.tsx:29
+msgid "Customise and expand."
+msgstr "Personalizează și extinde."
+
+#: apps/marketing/src/components/(marketing)/footer.tsx:38
+msgid "Design"
+msgstr "Design"
+
+#: apps/marketing/src/app/(marketing)/pricing/page.tsx:44
+msgid "Designed for every stage of your journey."
+msgstr "Proiectat pentru fiecare etapă a călătoriei tale."
+
+#: apps/marketing/src/components/(marketing)/carousel.tsx:37
+msgid "Direct Link"
+msgstr "Link Direct"
+
+#: apps/marketing/src/app/(marketing)/pricing/page.tsx:181
+msgid "Documenso is a community effort to create an open and vibrant ecosystem around a tool, everybody is free to use and adapt. By being truly open we want to create trusted infrastructure for the future of the internet."
+msgstr "Documenso este un efort al comunității de a crea un ecosistem deschis și vibrant în jurul unui instrument, pe care toți îl pot folosi și adapta gratuit. Fiind cu adevărat deschiși, dorim să creăm o infrastructură de încredere pentru viitorul internetului."
```

Comment:
"Documenso este un efort al comunității de a crea un ecosistem deschis și vibrant în jurul unui instrument, pe care toți îl pot folosi și adapta gratuit. Fiind cu adevărat `transparenti`, dorim să creăm o infrastructură de încredere pentru viitorul internetului."